packages/system/src/Report/api/api.ts

```diff
@@ -31,11 +31,11 @@ export const getReportQueries = (sdk: Sdk, storeId: string) => ({
         reportId,
         storeId,
       });
-      if (result.printReport.__typename === 'PrintReportNode') {
+      if (result?.printReport.__typename === 'PrintReportNode') {
         return result.printReport.fileId;
       }
 
-      throw new Error('Report not found');
+      throw new Error('Unable to print report');
```

Comment:
ok for now. ideally this should be translated.. but that's a bit hard in the api. fairly unlikely to be shown